src/pytest_merge_xml/plugin.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
+from typing import Dict
+
+from _pytest.reports import TestReport
+
+from pytest_merge_xml.merge_xml import modify_xml
+
+
+class XMLReport:
```

Comment:
```suggestion
class XMLReportPlugin:
```

Also XMLReport does not really conveys what the class means, how about `XMLReportMergerPlugin`, or something like that?